Projects/App/Sources/Screens/OnboardingScreen/WebOnboarding.swift

```diff
@@ -154,9 +134,33 @@ extension WebOnboarding: Presentable {
 
             viewController.present(alert)
         }
-
+        
         loadWebOnboarding()
 
-        return (viewController, bag)
+        return (viewController, Future { completion in
+            bag += webView.signal(for: \.url).map { url in
+                let urlString = String(describing: url)
+
+                if urlString.contains("connect-payment") {
+                    completion(.success)
+                }
+            }
+            
+            return bag
+        })
+    }
+}
+
+internal extension Environment {
+    var baseUrl: String {
```

Comment:
Sweet, should this maybe return a URL type instead?

Reply:
The url constructor returns a `url` this is just the base url -> Otherwise you need to create components etc.